pom.xml

```diff
@@ -54,21 +55,21 @@
     </dependency>
 
     <dependency>
-      <groupId>org.codehaus.staxmate</groupId>
+      <groupId>com.fasterxml.staxmate</groupId>
       <artifactId>staxmate</artifactId>
-      <version>2.0.1</version>
+      <version>2.4.1</version>
     </dependency>
 
     <dependency>
       <groupId>commons-io</groupId>
       <artifactId>commons-io</artifactId>
-      <version>2.6</version>
+      <version>2.16.1</version>
```

Comment:
So it stays I guess 😅

Reply:
No way we can get rid of the BOM problem in Java. It has been solved 20 years ago in .NET that it rarely gets into your way. 
Java in 2024: Still no solution